luminance-webgl/src/webgl2/state.rs

```diff
@@ -800,6 +810,24 @@ fn get_ctx_face_culling_mode(
   }
 }
 
+fn load_webgl_extensions(ctx: &mut WebGl2RenderingContext) -> Result<(), StateQueryError> {
+  let required_extensions = vec!["OES_texture_float_linear", "EXT_color_buffer_float"];
```

Comment:
No need to allocate / for a `Vec` here. Simply use an array.

Reply:
Done.